c7n/resources/rds.py

```diff
@@ -209,6 +210,44 @@ def __call__(self, rdb):
             self.default_vpc = vpcs.pop()
         return vpc_id == self.default_vpc and True or False
 
+@filters.register('kms-alias')
+class KmsKeyAlias(ValueFilter):
+
```

Comment:
same comments from the ebs kms key alias.. given these are identical bodies, rather than copying and pasting, having a common base class (in this case it looks like you don't even need a subclass) in kms.py as ResourceKmsKeyAlias and then just register it as the filters against ebs, rds would reduce the duplication.